containers/run-proxy-with-services.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+set -x
+
+CURDIR=$(dirname $(realpath $0))
+CONFIG_DIR="$CURDIR/proxy-config"
+SQUID_CACHE_DIR="$CURDIR/proxy-squid-cache"
+RHN_CACHE_DIR="$CURDIR/proxy-rhn-cache"
+TFTPBOOT_DIR="$CURDIR/proxy-tftpboot"
+
+export REGISTRY=registry.tf.local
+
+# HACK: overcome the inavailability of avahi in containers
+export ADD_HOST=server.tf.local:192.168.122.254
+export ADD_CLIENT=client.tf.local:192.168.122.89
+
+IMAGES=(proxy-ssh proxy-httpd proxy-salt-broker proxy-squid proxy-tftpd)
+
+for image in "${IMAGES[@]}"
+do
+  podman pull --tls-verify=false $REGISTRY/$image
+done
+
+###
+# POD
+###
+podman pod create --name proxy-pod \
+  --publish 22:22 \
+  --publish 8080:8080 \
+  --publish 443:443 \
+  --publish 4505:4505 \
+  --publish 4506:4506 \
```

Comment:
missing `--publish 69:69/udp` for tftp. Also do you intentionally don't specify udp and tcp ports?